<!DOCTYPE html>
<html lang="en-US">
<head>

	<title>OpenWrt Forum Archive</title>

	<meta charset="UTF-8">

	<meta http-equiv="X-UA-Compatible" content="IE=edge">

	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<link rel="stylesheet" href="assets/css/common.css">

</head>
<body>

<div class="container">

<header class="main-header">
	<h1 class="logo"><a href="index.html"><img src="assets/img/logo.png" width="376" height="88" alt="OpenWrt Forum Archive"></a></h1>
</header>

<aside>
	<p>This is a read-only archive of the old OpenWrt forum. The current OpenWrt forum resides at <a href="https://forum.openwrt.org/">https://forum.openwrt.org/</a>.</p>
	<p class="minor">In May 2018, the OpenWrt forum suffered a total data loss. This archive is an effort to restore and make available as much content as possible. Content may be missing or not representing the latest edited version.</p>
</aside>

<main>
	<header>
		<h1><span class="minor">Topic:</span> Buildbot broken?</h1>
	</header>
	<div class="notice minor">
		<p>
			The content of this topic has been archived
							on 22 Apr 2018.
										There are no obvious gaps in this topic, but there may still be some posts missing at the end.
					</p>
	</div>

	<div class="pagination"><div class="pagination-number">Page 1 of 1</div><nav><ul><li class="pagination-current"><span>1</span></li></ul></nav></div>
			
		
		
			<article class="post" id="p272996">
				<div class="post-metadata">
					<div class="post-num">Post #1</div>
					<div class="post-author">JimWright</div>
					<div class="post-datetime">
						18 Apr 2015, 23:43					</div>
				</div>
				<div class="post-content content">
					<p>Lots of failures the last three days on multiple builds:</p><br /><br /><p>Traceback (most recent call last):<br />&nbsp; File &quot;/usr/lib/python2.7/dist-packages/buildbot/process/buildstep.py&quot;, line 545, in _startStep_2<br />&nbsp; &nbsp; doStep.addCallback(self._startStep_3)<br />&nbsp; File &quot;/usr/lib/python2.7/dist-packages/twisted/internet/defer.py&quot;, line 298, in addCallback<br />&nbsp; &nbsp; callbackKeywords=kw)<br />&nbsp; File &quot;/usr/lib/python2.7/dist-packages/twisted/internet/defer.py&quot;, line 287, in addCallbacks<br />&nbsp; &nbsp; self._runCallbacks()<br />&nbsp; File &quot;/usr/lib/python2.7/dist-packages/twisted/internet/defer.py&quot;, line 545, in _runCallbacks<br />&nbsp; &nbsp; current.result = callback(current.result, *args, **kw)<br />--- &lt;exception caught here&gt; ---<br />&nbsp; File &quot;/usr/lib/python2.7/dist-packages/buildbot/process/buildstep.py&quot;, line 551, in _startStep_3<br />&nbsp; &nbsp; if self.start() == SKIPPED:<br />&nbsp; File &quot;/usr/lib/python2.7/dist-packages/buildbot/steps/master.py&quot;, line 112, in start<br />&nbsp; &nbsp; path=self.path, usePTY=self.usePTY, env=env )<br />&nbsp; File &quot;/usr/lib/python2.7/dist-packages/twisted/internet/posixbase.py&quot;, line 337, in spawnProcess<br />&nbsp; &nbsp; processProtocol, uid, gid, childFDs)<br />&nbsp; File &quot;/usr/lib/python2.7/dist-packages/twisted/internet/process.py&quot;, line 683, in __init__<br />&nbsp; &nbsp; self._fork(path, uid, gid, executable, args, environment, fdmap=fdmap)<br />&nbsp; File &quot;/usr/lib/python2.7/dist-packages/twisted/internet/process.py&quot;, line 390, in _fork<br />&nbsp; &nbsp; self.pid = os.fork()<br />exceptions.OSError: [Errno 12] Cannot allocate memory</p>									</div>
			</article>

			
		
	
			<div class="notice minor">
			<p>The discussion might have continued from here.</p>
		</div>
	
	<div class="pagination"><div class="pagination-number">Page 1 of 1</div><nav><ul><li class="pagination-current"><span>1</span></li></ul></nav></div>
</main>

</div>


<!-- Created in a hurry and not indicative of usual code quality. Here's a number: 0 -->

</body>
</html>